clients/src/test/java/org/apache/kafka/clients/consumer/internals/ConsumerCoordinatorTest.java

```diff
@@ -3206,14 +3208,21 @@ public void testRefreshOffsetUnknownTopicOrPartition() {
         assertThrows(KafkaException.class, () -> coordinator.initWithCommittedOffsetsIfNeeded(time.timer(Long.MAX_VALUE)));
     }
 
-    @Test
-    public void testRefreshOffsetNotCoordinatorForConsumer() {
+    @ParameterizedTest
+    @CsvSource({
+        "NOT_COORDINATOR, true",
+        "COORDINATOR_NOT_AVAILABLE, true",
+        "NETWORK_EXCEPTION, false",
+    })
+    public void testRefreshOffsetRetriableErrorCoordinatorLookup(Errors error, boolean expectCoordinatorRelookup) {
```

Comment:
now that we have parametrized this test, could we simplify and remove the existing `testRefreshOffsetLoadInProgress`, and just add another param here for `COORDINATOR_LOAD_IN_PROGRESS` ?

Reply:
Thanks for catching that, I'll factor it out.